### ISSUE TITLE
Suggestion/Request: Alert Dialog styling?

### ISSUE THREAD

**Issue by Excigma** [muckSponge/MaterialFox#188]
I thought dialogs were separate from the Firefox window somehow and could not be styled, and didn't really think about it, until I checked with the browser toolbox, and realized it is possible

Firefox Alert:
![image](https://user-images.githubusercontent.com/30280397/68541995-b40b1d00-040b-11ea-8568-3640c35f4a11.png)

Chrome's Alert:
![image](https://user-images.githubusercontent.com/30280397/68542001-cab17400-040b-11ea-839a-ef2e83963e34.png)

**Comment by Excigma** [comment]
Suddenly thought of it when Sorting by new and finding this:
https://www.reddit.com/r/FirefoxCSS/comments/ds68o8/any_chance_to_make_the_download_dialog_window_dark/
(May be interesting too?)

**Comment by muckSponge** [comment]
Yeah I'll add this to my to do list. I think we can do a bit better than Chrome here and use actual Material spec elements. Dialog boxes are reasonably well covered in the spec. Of coarse once we go down this road we'll have to do all the content UI (about: pages).
